packages/mui-base/src/ListboxUnstyled/useControllableReducer.test.tsx

```diff
@@ -53,25 +54,28 @@ describe('useControllableReducer', () => {
       });
 
       const actionToDispatch = { type: ActionTypes.setValue as const, value: 'b', event: null };
+      const props: UseListboxPropsWithDefaults<string> = {
```

Comment:
Can we extract this above and define it only once?

Reply:
I did this to make the tests more self-sufficient (it's easier to read when you have everything  nearby). I don't think repetition is a problem is tests.